components/modals/CreateCampaignStages/CampaignStageOne.tsx

```diff
@@ -36,11 +61,46 @@ export const CampaignStepOne = (
         </View>
         <View>
           <Text>Campaign Objective</Text>
-          <TextInput
-            style={styles.inputcampaign1}
-            value={campaignObjective}
-            onChangeText={setCampaignObjective}
-          />
+          <View
+            style={{
+              flexDirection: "row",
+              justifyContent: "space-between",
+              alignItems: "center",
+              width: "100%",
+              gap: 10,
+              position: "relative",
+            }}
+          >
+            <TextInput
+              style={styles.inputcampaign1}
+              value={campaignObjective}
+              multiline
+              numberOfLines={6}
+              onChangeText={setCampaignObjective}
+            />
+            <TouchableOpacity
```

Comment:
Let's not use TouchableOpacity much now onwards, It looks like it is deprecated or not robust -
<img width="835" alt="Screenshot 2024-12-03 at 11 52 36 AM" src="https://github.com/user-attachments/assets/113ba88a-cf88-450d-86b7-2db7d15d2a0f">